scan/lib/scan/options.rb

```diff
@@ -311,6 +311,18 @@ def self.available_options
                                        UI.user_error!("File not found at path '#{File.expand_path(value)}'") unless File.exist?(value)
                                      end),
 
+        # build settings
+        FastlaneCore::ConfigItem.new(key: :app_name,
+                                    env_name: "SCAN_APP_NAME",
+                                    optional: true,
+                                    description: "App name to use in slack message and logfile name",
+                                    is_string: true),
+        FastlaneCore::ConfigItem.new(key: :deployment_target_version,
+                                    env_name: "SCAN_DEPLOYMENT_TARGET_VERSION",
+                                    optional: true,
+                                    description: "Target version of the app being build or tested. Used to filter out simulator version",
+                                    is_string: true),
+
```

Comment:
initial thought was to remove duplicate showbuildsettings.. but while making changes, realized if we pass `target version` and `app name` we can totally avoid calling showBuildSettings which Scan much faster .. 

> 

>

Reply:
Ahhh, that is interesting! Its cool how much time that saved you but these options are pretty specific 🤔  I definitely want you to save that extra time but I'm going to think if there is some other way instead of adding two more options 😇